docs/user/intelmq-api.rst

```diff
@@ -0,0 +1,112 @@
+.. 
+   SPDX-FileCopyrightText: 2020 Birger Schacht
+   SPDX-License-Identifier: AGPL-3.0-or-later
+
+###########
+Intelmq Api
+###########
+
+intelmq-api is a `hug <http://hug.rest>`_ based API for the `intelmq <https://github.com/certtools/intelmq/>`_ project.
+
+**********************************
+Installing and running intelmq-api
+**********************************
+
+``intelmq-api`` requires at least `Python 3.6 <https://www.python.org/downloads/release/python-360/>`_.
+
+Install ``intelmq-api`` using your preferred package installation mechanism (``pip``, ``apt``, ``yum``, ``dnf``...).
+
+* ``pip install intelmq-api``
+* ``apt install python3-intelmq-api``
+* ``yum install intelmq-api``
+* ``zypper install intelmq-api``
```

Comment:
That's only possible if the intelmq repo is used.

Reply:
ACK, I've removed the list of installation commands and added a paragraph about the repository.